python/tests/global/test_formatting.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import os
+
+import pytest
+
+
+@pytest.mark.skipif(sys.version_info < (3, 6), reason="requires Python3")
+def test_code_style():
+    from pathlib import Path
+    import black
+    from click.testing import CliRunner
+
+    root = str(Path(__file__).parent.parent.parent)
+
+    runner = CliRunner()
+    resp = runner.invoke(
+        black.main,
+        [
+            "--check",
+            os.path.join(root, "job_runner"),
```

Comment:
I would prefer we instead used the `--exclude` flag and explicitly removed all current files not adhering to black formatting. Then new files would immediately be forced to follow the formatting rules.

Reply:
On second thoughts the workaround is causing problems of it's own, so a very explicit regex might be a better solution.